docs/getting-started/installation.rst

```diff
@@ -31,9 +31,9 @@ following commands in your terminal:
 The optional commands create an isolated conda environment called "*pmp*"
 within which you can safely work without breaking other python installations
 or running into python package incompatibilities. For more on working with
-conda environments, see `*Managing environments*`_ in the Conda docs.
+conda environments, see `**Managing environments**`_ in the Conda docs.
 
-.. _`*Managing environments*`: https://conda-forge.org/
+.. _`**Managing environments**`: https://conda-forge.org/
```

Comment:
```suggestion
.. _`Managing environments`: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html
```

This had the wrong link - whoops! Explicit emphasis is unnecessary, I think - the link alone emphasises the text.